roles/lokole/tasks/install.yml

```diff
@@ -23,24 +23,14 @@
       - wvdial
     state: present
 
-- name: "workarounds for older flask version used"
-  pip:
-    name: "{{ item.name }}"
-    version: "{{ item.version }}"
-    virtualenv: "{{ lokole_venv }}"
-    virtualenv_command: python3 -m venv "{{ lokole_venv }}"
-    extra_args: --no-cache-dir    # To avoid caching issues e.g. soon after new releases hit https://pypi.org/project/opwen-email-client/
-  with_items:
-    - { name: 'itsdangerous', version: '2.0.1' }
-    - { name: 'Jinja2', version: '3.0.3' }
-
 
 # For development purposes -- To install Lokole from a given commit, add the
 # following line to roles/lokole/defaults/main.yml:
 # lokole_commit: <git_commit_id>
+# TODO: revert back to main remo after upstream PR is merged
 - name: "OPTIONAL: pip install opwen_email_client (Lokole, git commit {{ lokole_commit }}) from GitHub to {{ lokole_venv }}, if lokole_commit is defined"
   pip:
-    name: "git+https://github.com/ascoderu/lokole.git@{{ lokole_commit }}#egg=opwen_email_client"
+    name: "git+https://github.com/mabuelhagag/lokole.git@{{ lokole_commit }}#egg=opwen_email_client"
```

Comment:
Temporary during testing, right?

(Presumably account 'mabuelhagag' will be changed back to 'ascoderu' later?)

Reply:
Might have to drop the 'git+' part and tinker with the rest of the string to get it right, but that is the general idea.

Reply:
One could move the definition from hardcoded to softcoded and use local_vars to override.
  
pip:
    name: "{{  lokole_repo  }}" 

then define in defaults/main.yml
lokole_repo: git+https://github.com/ascoderu/lokole.git@{{ lokole_commit }}#egg=opwen_email_client
and then use the override in local_vars.yml
lokole_repo: git+https://github.com/mabuelhagag/lokole.git@{{ lokole_commit }}#egg=opwen_email_client

Reply:
Check if https://github.com/iiab/iiab/pull/3276/commits/191e7577c3ea46d20e2a64d9c331335475396042 does it

Reply:
I have updated the PR description accordingly